src/language-html/utils.js

```diff
@@ -376,7 +376,10 @@ function inferScriptParser(node) {
       return "markdown";
     }
 
-    if (node.attrMap.type === "application/ld+json") {
+    if (
+      node.attrMap.type.substr(-4) === "json" ||
```

Comment:
```suggestion
      node.attrMap.type.endsWith("json") ||
```

Reply:
As I said in my comment above, I didn't use `endsWith` because it's not available in Node 6.4.

Reply:
Am I missing something? :thinking: 

```
❯ node --version
v6.4.0

❯ node
> "".endsWith
[Function: endsWith]
> 
```

Reply:
Looks like I misread the node.green table. I could've sworn it said it was unsupported! I used `endsWith` initially but changed it after checking.